src/bp/core/routers/admin/bots.ts

```diff
@@ -180,6 +180,25 @@ export class BotsRouter extends CustomRouter {
       })
     )
 
+    router.post(
+      '/:botId/import',
+      this.asyncMiddleware(async (req, res) => {
+        let body = ''
+        req.on('data', chunk => {
+          body += chunk.toString()
```

Comment:
wouldn't it be better to use a BufferArray here instead of converting to string and converting it back to a buffer at the end?

Reply:
@slvnperron i am taking a look into the alternatives, did you mean ArrayBuffer?

Reply:
You does not need to convert chunk to string. Save it as [ArrayBuffer](https://stackoverflow.com/questions/11554006/javascript-arraybuffer-whats-it-for/11555049)